src/ocm/services/OperatorsService.tsx

```diff
@@ -0,0 +1,40 @@
+import {
+  V2ClusterUpdateParams,
+  OPERATOR_NAME_CNV,
+  OPERATOR_NAME_ODF,
+  OPERATOR_NAME_OCS,
+  OPERATOR_NAME_LSO,
+  HostDiscoveryValues,
+  Cluster,
+} from '../../common';
+import { getOlmOperatorCreateParamsByName } from '../components/clusters/utils';
+const OperatorsService = {
+  setOLMOperators(
+    params: V2ClusterUpdateParams,
+    values: HostDiscoveryValues,
```

Comment:
Thanks for the clean-up. However, I see a few places that are missing:

- OperatorService should use `OperatorsValues` and not `HostDiscoveryValues`
- `HostDiscoveryValues` should not include  `useExtraDisksForLocalStorage` or `useContainerNativeVirtualization`.
- The call to `setOLMOperators` in `Operators` needs to be updated with the last parameter, and it needs to be removed from `HostDiscovery` 
- `getHostDiscoveryInitialValues` should not include the moved fields.

Reply:
I believe I did some of those things in the past, but I lost it on one of the rebasing, I guess
I'll do it now